static/report/js/report.js

```diff
@@ -448,7 +448,13 @@
     options.maxInsulinValue = maxInsulinValue;
     options.maxCarbsValue = maxCarbsValue;
 
-
+    datastorage.treatments = [];
+    datastorage.combobolusTreatments = [];
+    for (var day in daystoshow) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1498320474/issues/source?bid=2452379&fileBranchId=2998783#l453)